src/cmd/health.js

```diff
@@ -108,14 +131,69 @@ async function get_service_memory_usage() {
 }
 
 
+function get_root_fs_context() {
+  return {
+      uid: process.getuid(),
+      gid: process.getgid(),
+      warn_threshold_ms: config.NSFS_WARN_THRESHOLD_MS,
+  };
+}
+
+async function get_bucket_storage_status(config_root) {
+  return await get_storage_status(config_root, 'bucket');
+}
+
+async function get_account_storage_status(config_root) {
+  return await get_storage_status(config_root, 'account');
+}
+
+async function get_storage_status(config_root, type) {
+  const fs_context = get_root_fs_context();
+  const entries = await nb_native().fs.readdir(fs_context, get_config_path(config_root, type));
+  const config_files = entries.filter(entree => !native_fs_utils.isDirectory(entree) && entree.name.endsWith('.json'));
+  const invalid_storages = [];
+  try {
+    for (const config_file of config_files) {
+      const config_file_path = path.join(get_config_path(config_root, type), config_file.name);
+      const raw_data = await fs.promises.readFile(config_file_path);
+      const config_data = JSON.parse(raw_data.toString());
+      if (config_data) {
+        const storage_path = type === 'bucket' ? config_data.path : config_data.nsfs_account_config.new_buckets_path;
+        const dir_stat = await fs_utils.file_exists(storage_path);
+        if (!dir_stat) {
+            console.error(`Error: Storage path should be a valid dir path`, storage_path);
+            const invalid_storage = {
+              name: config_data.name,
+              storage_path: config_data.path ? config_data.path : config_data.nsfs_account_config.new_buckets_path,
+            };
+            invalid_storages.push(invalid_storage);
+        }
+      } else {
+          console.error('Could not parse config file: ', config_file);
+      }
+    }
+  } catch (err) {
+      console.error('Error while accessing the storage: ', err);
+  }
+  return invalid_storages;
+}
+
+function get_config_path(config_root, type) {
+  return path.join(config_root, type === 'bucket' ? '/buckets' : '/accounts');
+}
+
+
 async function main(argv = minimist(process.argv.slice(2))) {
   try {
 
     if (argv.help || argv.h) return print_usage();
-
+    const config_root = argv.config_root ? String(argv.config_root) : config.NSFS_NC_DEFAULT_CONF_DIR;
+    // disable console log to avoid unwanted logs in console.
+    disable_console_log();
```

Comment:
I don't think we should do it like that, we maybe don't want to print it but it's useful to have the logs of this health script somewhere else

Reply:
@romayalon Health script prints unwanted logs from nb_native.js, cant use debug_module there because of the module conflict. WDYT?

Reply:
@romayalon All the logs related to the health check are pushed to `syslog`,  console log comes only from `nb_native.js` and I don't think it contains any useful info for health script.

Reply:
as we discussed today, let's keep it this way for this PR and on the next PR change it as Guy said for both manage_nsfs.js and health.js